### ISSUE TITLE
Branch coverage reported incorrectly

### ISSUE THREAD

**Issue by joshmanderson** [jenkinsci/cobertura-plugin#83]
`branch-rate` output in the coverage xml file is `0.1721`, yet the report states that the "Conditionals's health is 13.12" (and then fails the build because the unhealthy threshold is `15.0`). Upon closer inspection of the branch coverage, many files are slightly off (reports that 2 branches were missed even though they weren't), and some files are *way* off (reports that there was a maximum of 2 branches in the file, even though there is actually 39).

This issue occurs using versions `1.12` and `1.10` of the plugin.

Something to note is that coverage reporting is currently working for a different project, and the main differences are that the project which is **failing** was bootstrapped using [create-react-app](https://github.com/facebookincubator/create-react-app) and is using version `20.0.4` of [jest](https://facebook.github.io/jest/). Whereas the project where coverage reporting is **working** was not bootstrapped (created from scratch), and is using version `21.1.0` of jest. Note that both projects are using the same version of [istanbul-reports](https://github.com/istanbuljs/istanbuljs/tree/master/packages/istanbul-reports) (`1.1.2`) – in case that matters.

I thought initially that the coverage wasn't being collected properly, however the html and xml files generated seem to have the correct coverage values in them, it's only the branch values reported by the plugin which are incorrect.

To summarise the problem:
- Correct coverage values are output to the coverage xml file
- Incorrect branch/conditional coverage values are reported by the plugin

Are you aware of any version compatibility issues that could be occurring here such that the xml file is read incorrectly?
